app/constants/view.js

```diff
@@ -42,7 +42,11 @@ const ViewTypes = keyMirror({
     SET_LAST_CHANNEL_FOR_TEAM: null,
 
     GITLAB: null,
-    SAML: null
+    SAML: null,
+
+    INCREASE_POST_VISIBILITY: null,
```

Comment:
out of scope of this PR, but we'll be using FOCUSED POST for search and for permalinks